### ISSUE TITLE
Parent relationship not displaying records for child works

### ISSUE THREAD

**Issue by HKativa** [sciencehistory/chf-sufia#1112]
Scenario: You are an admin. From your "My works" page, you select a work to edit. Under the "Relationships" tab, you relate several child works to this parent work using filenames in the "search for a work" feature. Upon saving the record, the child works display in the parent work, but the parent relationship does not display in the records for the child works.

Two documented occurrences of this bug:
1) Parent work ID: c247ds983
Child work IDs
xw42n8833
jq085k773
9c67wn818
fx719n30c
8623hz583
0c483k35d
2n49t2644
1544bp91n
6q182k958
st74cr33q
mp48sd69w
th83m027m
2j62s586n
08612p53t

2) Parent work ID: m900nv29n
Child work IDs:
dz010r02k
5999n4223
gt54kn78z
1c18dg815
8336h283c
pv63g1291
zs25x934s
hm50ts54g
vt150k081
m039k600b
3n203z946
8336h284n
2j62s5886
td96k3530
sb397923d
df65v857q
w0892b85p
ht24wk37kj
d472x39w
zw12z615t

**Comment by HKativa** [comment]
@eddierubeiz FYI this may be related to the software error bug reported under #1105

**Comment by HKativa** [comment]
For example of what the parent-child relationship looks like when displaying properly, see parent work ID dn39x222c and child work ID nv935360n

**Comment by jrochkind** [comment]
updating the solr index on the _parents_ involved seems to resolve the issue. 

```
irb(main):007:0> parent_work = GenericWork.find("m900nv29n")
=> #<GenericWork id: "m900nv29n", head: [#<ActiveTriples::Resource:0x4f7a5a4(#<ActiveTriples::Resource:0x00000009ef4b48>)>], tail: [#<ActiveTriples::Resource:0x4f48a7c(#<ActiveTriples::Resource:0x00000009e914f8>)>], depositor: "hkativa@sciencehistory.org", title: ["The microscopy of drinking water"], date_uploaded: "2018-06-19 15:30:02", date_modified: "2018-06-19 15:45:22", state: #<ActiveTriples::Resource:0x4f3b610(#<ActiveTriples::Resource:0x00000009e76c20>)>, owner: nil, proxy_depositor: nil, on_behalf_of: nil, arkivo_checksum: nil, label: nil, relative_path: nil, import_url: nil, part_of: [#<ActiveTriples::Resource:0x4f283d0(#<ActiveTriples::Resource:0x00000009e507a0>)>], description: [], date_created: [], subject: ["Water--Analysis", "Microscopy", "Microscopes"], language: [], identifier: ["bib-b1015936"], based_near: [], related_url: [], bibliographic_citation: [], source: [], creator: [], after: [], artist: [], author: ["Whipple, George Chandler, 1866-1924"], addressee: [], creator_of_work: [], contributor: [], engraver: [], interviewee: [], interviewer: [], manufacturer: [], photographer: [], printer: [], printer_of_plates: [], publisher: ["John Wiley & Sons"], place_of_interview: [], place_of_manufacture: [], place_of_publication: [], place_of_creation: [], additional_title: [], admin_note: [], credit_line: ["Courtesy of Science History Institute"], division: "Library", exhibition: [], file_creator: "Muhlin, Jay", genre_string: ["Illustrations", "Publications"], extent: ["24 cm H"], medium: [], physical_container: "", resource_type: ["Text", "Image"], rights: ["http://creativecommons.org/publicdomain/mark/1.0/"], rights_holder: "", series_arrangement: [], access_control_id: "66658a7d-f2d6-49ee-b4d3-891abb1484db", representative_id: "b8515p16f", thumbnail_id: "b8515p16f", admin_set_id: "admin_set/default", embargo_id: nil, lease_id: nil, date_of_work_ids: ["7beebc78-49db-4692-92ed-6271a047b98f"], inscription_ids: [], additional_credit_ids: []>
irb(main):008:0> parent_work.update_index
=> {"responseHeader"=>{"status"=>0, "QTime"=>674}}
irb(main):009:0>
```

All children are now displaying correctly. 

I am not sure why the parents did not get updated correctly in the first place in the Solr index. 

It could be related to #1105. And one or both of those could be related to the new ingest code we deployed to handle S3. 

This is, alas, a very fragile stack. 

I am going to close this for now because the immediate issue is resolved, but the link will show up on #1105 to remind us that this may be related.
